runtime/onert/core/src/exec/Execution.cc

```diff
@@ -126,21 +126,22 @@ void Execution::setOutputType(const ir::IOIndex &index, const ir::TypeInfo &type
   _ctx.shape_updated = true;
 }
 
-void Execution::execute()
+void Execution::execute(const ExecutionOptions &options)
```

Comment:
I have a question.
Is it difficult to set `ExecutionOptions`  only when `Execution` is created? It seems that `ExecutionOptions` is not changed, but is set whenever `execute()` is called.

Reply:
`Execution` is created on `nnfw_prepare()` API, but `ExecutionOptions` can be updated by `nnfw_set_execute_config` APIs. 

To change to set `ExectionOptions` on `Execution` creation, we need to update I/O buffer setting and tensorinfo getter/setter implementation, then we can create `Execution` when `nnfw_run()` is called. It requires many implementation changes.

Reply:
I'll try.